### ISSUE TITLE
Unhandled method textDocument/completion

### ISSUE THREAD

**Issue by vwchu** [znck/grammarly#6]
I keep get this error message like this in my console:

```
[Error - 1:20:21 a.m.] Request textDocument/completion failed.
  Message: Unhandled method textDocument/completion
  Code: -32601 
```

**Comment by vwchu** [comment]
Also get these messages in the console:
```
tag p has no close,is self-close? use class SelfCloseTag
Grammarly connection terminated due to error: { error: 'runtime_error', severity: 'error', action: 'error', id: 37 }
```

**Comment by afonsojramos** [comment]
From a day where everything was working, I am now also getting this issue

**Comment by jck-ccl** [comment]
I'm seeing this too

**Comment by badetitou** [comment]
I have the problem too

```
 {
  error: 'cannot_find_synonym',
  severity: 'info',
  action: 'error',
  id: 211
}
```

**Comment by wilburpowery** [comment]
I'm premium and also getting the problem FWIW!

**Comment by znck** [comment]
Fixed in 1ecb519901206268a73050df8aea4181ab184fc9
